website/common/locales/en/groups.json

```diff
@@ -468,5 +468,9 @@
   "howToRequireApprovalDesc2": "Group Leaders and Managers can approve completed Tasks directly from the Task Board or from the Notifications panel.",
   "whatIsGroupManager": "What is a Group Manager?",
   "whatIsGroupManagerDesc": "A Group Manager is a user role that do not have access to the group's billing details, but can create, assign, and approve shared Tasks for the Group's members. Promote Group Managers from the Group’s member list.",
-  "goToTaskBoard": "Go to Task Board"
+  "goToTaskBoard": "Go to Task Board",
+  
+  "approvalNotRequired": "Approval not required",
+  "messagedFlaggedNotHidden": "Message flagged once, not hidden",
+  "messageHidden": "Message hidden"
```

Comment:
I can move the server side one. The other two are on the client so we don't have access to that lib there.

Reply:
For those two client-side strings, it might be best to have them hard-coded into the vue files. That would save the translators translating them unnecessarily and would slightly reduce the volume in Transifex. Alternatively, perhaps we could have a `doNotTranslate.json` locales file that's never uploaded to Transifex.